data/config.yaml

```diff
@@ -3,25 +3,25 @@
 knobs:
     # Note: normally you don't set these... here just to show how to
     # devel and debug
-    # log_to_console: false
```

Comment:
please create data/config.yaml.sirkingchase  and leave this one as is. It is here mostly as an example and should be complete but simple.